weblate/trans/validators.py

```diff
@@ -16,30 +16,30 @@
 
 
 def validate_filemask(val):
-    """Validate that filemask contains *."""
+    """Validate that the filemask contains *."""
     if "*" not in val:
         raise ValidationError(
             gettext("File mask does not contain * as a language placeholder!")
         )
 
 
 def validate_autoaccept(val):
-    """Validate correct value for autoaccept."""
+    """Validate correct value for auto-accept."""
     if val == 1:
         raise ValidationError(
             gettext(
-                "A value of 1 is not allowed for autoaccept as "
+                "A value of 1 is not allowed for auto-accept as "
                 "it would permit users to vote on their own suggestions."
             )
         )
 
 
 def validate_check_flags(val):
-    """Validate check influencing flags."""
+    """Validate check-influencing flags."""
     try:
         flags = Flags(val)
     except (ParseException, re.error) as error:
-        raise ValidationError(gettext("Failed to parse flags: %s") % error)
+        raise ValidationError(gettext("Could not parse flags: %s") % error)
```

Comment:
Can you please avoid stacking random changes to a documentation pull request? This is a useful change, but should be done in a separate pull request, and cover all such strings. I've just done this at https://github.com/WeblateOrg/weblate/pull/9575

Reply:
Neither option is ideal really. I care for end-product and coverage, not percentage-wise total flaws.

Separate PRs would remove my only way of ensuring quality, which is knowing what files got reworked (in their entirety) based on the last time I touched anything.

I want to know where bugs aren't, according to my ability. Per-reviewer quality is my way, and it doesn't matter if I can't know what people I trust to do the work think.
If those people, which is me in some capacity are not part of what goes in at an intermediary stage, looks like that is as good as it gets once stuff is merged. That is as far as I am concerned. It can go in, but it can't stay there.
My process is to notice an error, and then to fix that file.

If I got write-access to the glossary, I could add incorrect or undesired words.
This in turn could trigger a bug-report/to-be-done queue of sorts, as it is a near endless list.
If I even attempted to make bug-reports for everything I find across the entire string-base I would never get done doing anything, and it would swarm the tracker. I have good memory of what should and is done at word-for-word level, but would lose critical oversight of what exists in a passable state that way.
Then whoever could discuss those glossary entries before they are accepted, and I don't even have to make the changes, or handle Git.
Maybe batch-suggestions would be a useful idea. I search for something, don't alter all of it, but instead adjust it to where it can be one-clicked by an admin.
F-Droid has a distinctive spell-checker triggered in the repo, but I much prefer it to be handled "in Weblate", and to also get input from all translators before going live.

I notice bugs, go check to see that I have a PR on the file, and disregard.

As an example "e.g." is used in the documentation (that I haven't fully translated so it is harder for me to know what strings are new), whereas "for example" is used in the software.
Add to this that suggestions are turned off for the "Legal" component, and it all gets a bit difficult for no real reason.